floor_generator/test/misc/extension/type_converter_element_extension_test.dart

```diff
@@ -37,19 +37,6 @@ void main() {
       expect(actual, equals({expected}));
     });
 
-    test('throws error when null in annotation', () async {
-      const typeConverterScope = TypeConverterScope.database;
-      final element = await '''
-        @TypeConverters(null)
-        abstract class Foo {}
-      '''
-          .asClassElement();
-
-      final actual = () => element.getTypeConverters(typeConverterScope);
-
-      expect(actual, throwsProcessorError());
-    });
```

Comment:
Hi, thanks for your input :D. 
The author and I spoke briefly about this offline and this is what I thought:
The TypeConverters annotation cannot take a null value anymore. (Which in my opinion makes sense, because if you have no type converters, you can just not use the annotation, so I expect there always to be at least one element in the list).
Therefore, with a null value, it won't compile so there is no need for a test if it already fails at compile time.
But of course, please correct me if this view is too limited.

Reply:
ok, got it
But in any case, if we're talking about the code generation phase rather than compile time, we need to be sure that this situation causes the correct error.
This was the main idea of this test

Reply:
Thanks for your input, I understand, lets get those tests back in and get them to expect the right error

Reply:
I put the test back and it expect the right error now.